### PR TITLE
FWF:3555-[Feature] Create custom inputbox

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3555
Issue Type: BUG/ FEATURE

# Changes
1.Added styles for FormInput component

### CODE REVIEW

File: forms-flow-theme/scss/_forms.scss [hunk 1]
```diff
@@ -223,4 +223,85 @@ select option:hover {
...
+    border:2px solid var(--ff-primary, #253DF4) ;
+    }
+
+    &::placeholder{
+      color: var(--ff-gray-400, #949B9E); 
+      font-family: var(--primary-font);
+      font-size: var(--font-size-sm);
+      font-style: normal;
+      font-weight: var(--font-weight-sm);
+      line-height: 120%;
```
Comment (Ajay-aot) [inline forms-flow-theme/scss/_forms.scss:268]: use line height variable here

File: forms-flow-theme/scss/_forms.scss [hunk 2]
```diff
@@ -223,4 +223,85 @@ select option:hover {
...
+    color:var(--ff-primary, #253DF4) ;
+  }
+
+  .custom-form-control-label{
+    color: var(--ff-gray-800, #303436);
+    font-family:var(--primary-font) ; //Roboto
+    font-size: var(--font-size-xs) ; //14px
+    font-style: normal;
+    font-weight: var(--font-weight-sm); //400
+    line-height: 120%;
```
Comment (Ajay-aot) [inline forms-flow-theme/scss/_forms.scss:291]: add variable here also

File: forms-flow-theme/scss/_forms.scss (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-theme/scss/_forms.scss:268]: updated

File: forms-flow-theme/scss/_forms.scss (hunk 2 above)
Comment (fahad-aot) [reply forms-flow-theme/scss/_forms.scss]: Updated.

File: forms-flow-theme/scss/_forms.scss [hunk 3]
```diff
@@ -223,4 +223,85 @@ select option:hover {
...
+    line-height: 120%; 
+  }
+
+  .custom-icon{
+    color: var(--ff-primary, #253DF4) !important;
+    z-index: 10;
+  }
+
+  .custom-feedback{
+    color: var(--sucess-fail-Red-000, #FF4242);
```
[the remark this answers is not in the document]
Comment (fahad-aot) [reply forms-flow-theme/scss/_forms.scss]: updated.

File: forms-flow-theme/scss/_forms.scss [hunk 4]
```diff
@@ -223,4 +223,91 @@ select option:hover {
...
+    border: 1px solid var(--ff-primary, #253DF4);
+
+    &:focus {
+    border:2px solid var(--ff-primary, #253DF4) ;
+    }
+
+    &::placeholder{
+      color: var(--ff-gray-400, #949B9E); 
+      font-family: var(--primary-font);
+      font-size: var(--font-size-sm);
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_forms.scss:265]: Remove 
font-family:var(--primary-font) ; //Roboto
font-style: normal;
everywhere. it is added in body

Comment (fahad-aot) [reply forms-flow-theme/scss/_forms.scss:264]: updated.

**Pull request merged**